terraform/infrastructure/etc/dev.tfvars

```diff
@@ -3,3 +3,4 @@ account_name = "dev"
 domain                = "api.record-locator.dev.national.nhs.uk"
 public_domain         = "internal-dev.api.service.nhs.uk"
 public_sandbox_domain = "internal-dev-sandbox.api.service.nhs.uk"
+log_retention_period  = 90
```

Comment:
90 days seems like a lot of logs for a dev env but happy to leave as is and review volume/cost over time.

Reply:
yeah thats what it was agreed on